app/controllers/pdl.py

```diff
@@ -93,6 +93,8 @@
 
 TITLE_II_START_YEAR = 2014
 TITLE_II_END_YEAR = 2023
+CROP_INSURANCE_START_YEAR = 2014
```

Comment:
Similar to my comment on the data ingestion PR, is it worthwhile to consider putting the years in environment variables so they can be updated without new code deployment?